bspump/mqtt_metrics/__init__.py

```diff
@@ -93,90 +101,95 @@ def __init__(self, app, service_name="bspump.MQTTService"):
         self.client.connect(self.host, int(self.port), 60)
         self.client.loop_start()
         self.connected = False
+        try:
+            self.max_count = int(asab.Config["MQTTMetrics"].get("max_count"))
+        except Exception:
+            L.warning(
+                "MQTTService: The max_count parameter is not set in the configuration file or is not an integer. Default value is 10000."
+            )
+            self.max_count = 10000
```

Comment:
```suggestion
            self.max_count = 100
```
If there are 10 000 events per second we don't want to slow down the pipeline by sending every single one. We want this number to be low.

Reply:
Oh based on your previous comment I though that you wanted to have more than 2000 but I guess we misunderstood